webapp/crashstats/supersearch/tests/test_views.py

```diff
@@ -105,209 +87,141 @@ def test_search_fields_metrics(self):
             ],
         )
 
-    def test_search_results(self):
+    def test_search_results(self, client, db, es_helper):
         BugAssociation.objects.create(
             bug_id=123456, signature="nsASDOMWindowEnumerator::GetNext()"
         )
 
-        def mocked_supersearch_get(**params):
-            assert "_columns" in params
-
-            if "WaterWolf" in params.get("product", []):
```

Comment:
These results were all mocked out and were wrong in a lots of ways. Also, when we go to update Elasticsearch, these tests would continue to pass even if the API changed which is deeply unhelpful.